src/app/app.component.ts

```diff
@@ -116,7 +117,9 @@ export class AppComponent {
     for (const initAction of APP_INITIALISATION_DEFAULTS.app_first_launch_actions) {
       switch (initAction.type) {
         case "template_popup":
-          await this.templateService.runStandaloneTemplate(initAction.value);
+          await this.templateService.runStandaloneTemplate(initAction.value, {
+            showCloseButton: false,
+          });
```

Comment:
minor tweaks made in future section to allow passing additional options to standalone templates (such as whether to include a close button or not)